.github/workflows/test-all-packages.yml

```diff
@@ -86,45 +91,47 @@ jobs:
       #- name: yarn test (everything)
       #  run: yarn ${{ steps.vars.outputs.test }}
       - name: yarn test (access-token)
-        run: cd packages/access-token && yarn ${{ steps.vars.outputs.test }}
+        run: cd packages/access-token && yarn ${{ steps.vars.outputs.test }} | tee -a _testoutput.txt
```

Comment:
please DRY this out. the `tee` only needs to be added once, up in `echo ::set-output name=test::${{ matrix.engine == 'xs' && 'test:xs' || 'test' }}`.

Related, does `test:xs` output in the same format? If not then you can use that conditional to tee only off plain `test`.

Reply:
> the `tee` only needs to be added once

Oh, good point!  So you mean:

```sh
          echo "::set-output name=test::${{ matrix.engine == 'xs' && 'test:xs' || 'test' }} | tee -a _testoutput.txt"
```

Reply:
yup, if `test:xs` outputs what's needed. I suspect it doesn't since it's our own test runner. If not then,
```
          echo "::set-output name=test::${{ matrix.engine == 'xs' && 'test:xs' || 'test | tee -a _testoutput.txt' }}"
```

Reply:
how does this work with https://github.com/Agoric/agoric-sdk/pull/6855/files/dc4df56b3308345cdcfc32c22e6c6a9feb78c43c#diff-2f93a8eb3e738e55500ff9c1dd529ca63a2e9ccfc6d01535730bf76dd15e9cb7R337

Reply:
Same. Any line that has `yarn ${{ steps.vars.outputs.test }}` shouldn't need modification. Just change the `set-output` directive that's specifying `steps.vars.outputs.test`.

Reply:
```
        run: cd packages/SwingSet && yarn ${{ steps.vars.outputs.test }} 'test/**/test-[S-Zs-z0-9]*.js' | tee -a _testoutput.txt
```

in this case, the tests outputs is before the qualifying subject for tests

Reply:
Shoot, I didn't notice that SwingSet breaks up one package across multiple jobs. Yeah in that case the repeated pipe is fine. If this data proves useful I expect we'll make `yarn test` produce it without any shell shenanigans

Reply:
Unfortunately, after some experimentation, GitHub forcibly escapes the pipe through all routes of ENV or output, so I could only template the output command; however, that is still an improvement here.